skrub/_reporting/_data/templates/tabs.css

```diff
@@ -0,0 +1,47 @@
+.tab {
```

Comment:
it doesn't come from anywhere, I wrote it. it's in a separate file only because I tried to divide them a bit depending on the parts of the report they apply to, to avoid getting some very large files. but I'll see if I can clean it up / reorganize it a bit. or we can put all the css in one file.